packages/pods-core/src/builtin/NextjsExportPod.ts

```diff
@@ -362,7 +362,24 @@ export class NextjsExportPod extends ExportPod<NextjsExportConfig> {
     const logo = ConfigUtils.getLogo(config);
     if (logo && !isWebUri(logo)) {
       const logoPath = path.join(wsRoot, logo);
-      fs.copySync(logoPath, path.join(siteAssetsDir, path.basename(logoPath)));
+      try {
+        const targetPath = path.join(siteAssetsDir, path.basename(logoPath));
+        await fs.copy(logoPath, targetPath);
+      } catch (err: any) {
+        // If the logo file was missing, that shouldn't crash the
+        // initialization. Warn the user and move on.
+        if (err?.code === "ENOENT") {
```

Comment:
Can we do `fs.pathExists(targetPath)` instead of try catch on ENOENT?

Reply:
That's less reliable, if the file got deleted between `fs.pathExists` and `fs.copy` then we would hit the same bug again. It would be a rare case, but checking before use is unreliable with file systems since it can change outside your control.

If you are more worried about maintainability or readability of the code, I can create a utility around this instead to make it clearer.